pages/subscribe/subscribe.css

```diff
@@ -9,8 +9,17 @@ body {
   margin: 1em auto;
 }
 
+#title {
+  font-size: 1.7em;
+}
+
+#subscribe-button {
+  margin-top: 1.139em;
```

Comment:
Can we avoid using special margins, and use flexbox alignment methods ?
Would markup like this work? with `align-self: center` on #subscribe-button 
```html
    <div class="flex">
      <h1 id="title" class="grow">Blog - Janitor community forum</h1>
      <button class="primary icon new icon-white" id="subscribe-button">Add Livemark</button>
    </div>
    <p id="description">Topics in the 'Blog' category Articles, and comments, for the Janitor blog. published posts are also on https://janitor.technology/blog/</p>
    <div>
      <input id="feed-url" type="text">
    </div>
    <div>
      <a class="icon view-source" id="view-source-button" href="view-source:https://discourse.janitor.technology/c/propaganda/blog.rss">View source</a>
    </div>
  ```

Reply:
I tried this and it didn't really work. I really not good at CSS.